codegen/gql_tristate_value/test/value_test.dart

```diff
@@ -2,33 +2,107 @@ import 'package:gql_tristate_value/gql_tristate_value.dart';
 import 'package:test/test.dart';
 
 void main() {
-  test("requireValue on absent throws", () {
-    final absent = const AbsentValue();
+  group(
+    'requireValue',
+    () {
+      test('on $AbsentValue throws', () {
```

Comment:
please remove the $ from the test strings.

I like static test names better if possible, dynamic ones make filtering by test name harder, and the test names generally become weird:

<img width="790" alt="Screenshot 2024-02-20 at 17 51 10" src="https://github.com/gql-dart/gql/assets/1313079/5ba94bc3-08f3-43cb-b456-5922f62d49c0">

Reply:
<img width="377" alt="Screenshot 2024-02-20 at 17 52 59" src="https://github.com/gql-dart/gql/assets/1313079/bd619320-2d05-4baa-9206-1f194057d82c">

Also breaks IntellIj "retun test" feature

Reply:
For sure `${Value.ofNullable}` should not be there. I forgot to check how it looks like in logs.
But personally I like to use e.g. `$AbsentValue`, because it's very helpful when refactoring the code. You don't have to remember about every place in the code where you have just a string with the class name.
But I changed all the names to static ones 👌